packages/squiggle-lang/scripts/lib.mjs

```diff
@@ -39,3 +39,26 @@ export const run = (src, { output, sampleCount } = {}) => {
     result.tag === "Error" ? result.value.toString() : ""
   );
 };
+
+export const runProject = (project, output) => {
```

Comment:
I'd suggest reimplementing `run` to call `runProject` to avoid copy-paste.

Also, second parameter should be an object for future extensibility, compatibility with `run` and passing in `sampleCount`.